packages/core/chrome/core-chrome-browser/src/project_navigation.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { ComponentType } from 'react';
+
+/** @internal */
+type AppId = string;
+
+/** @internal */
+type DeepLinkId = string;
+
+/** @internal */
+export type AppDeepLinkId = `${AppId}:${DeepLinkId}`;
+
+/** @public */
+export type ChromeProjectNavigationLink = AppId | AppDeepLinkId;
+
+/** @public */
+export interface ChromeProjectNavigationNode {
+  id?: string;
+  link?: ChromeProjectNavigationLink;
+  children?: ChromeProjectNavigationNode[];
+
+  title?: string;
+  icon?: string;
+}
+
+/** @public */
+export interface ChromeProjectNavigation {
+  navigationTree: ChromeProjectNavigationNode[];
```

Comment:
Can we have this include `platformConfig`, along the lines of what the [Shared UX navigation component](https://github.com/elastic/kibana/blob/[604a02f](https://github.com/elastic/kibana/commit/604a02f6a12765e703bd8a2d6623120b3c92ea25)/packages/shared-ux/chrome/navigation/types/index.ts#L62) expects?

Reply:
The `ChromeProjectNavigation ` is really just a placeholder for now that @Dosant created. It probably misses a few things (like also the link to the home page etc...). 

I'll set the proper interface in a following PR 👍